src/backend/expungeservice/record_summarizer.py

```diff
@@ -13,27 +15,30 @@ class RecordSummarizer:
     def summarize(record, questions: Dict[str, QuestionSummary]) -> RecordSummary:
         county_fines, county_filing_fees = RecordSummarizer._build_county_balances(record)
         eligible_charges_by_date = RecordSummarizer._build_eligible_charges_by_date(record)
+        eligible_nonconvictions_only = RecordSummarizer._build_eligible_nonconvictions_only(record.charges)
         return RecordSummary(
             record=record,
             questions=questions,
             eligible_charges_by_date=eligible_charges_by_date,
             total_charges=len(record.charges),
             county_fines=county_fines,
             county_filing_fees=county_filing_fees,
+            eligible_nonconvictions_only=eligible_nonconvictions_only,
```

Comment:
I feel sad about this variable name because to be fully accurate it needs to be true when the number of eligible convictions is zero. A more accurate name is `has_eligible_nonconvictions_and_no_eligible_convictions`. But that is probably too silly.

Reply:
What about `is_all_cases_feeless`